R/zchunk_LA106.ag_an_NetExp_FAO_R_C_Y.R

```diff
@@ -30,6 +30,9 @@ module_aglu_LA106.ag_an_NetExp_FAO_R_C_Y <- function(command, ...) {
 
     all_data <- list(...)[[1]]
 
+    Exp_t <- GCAM_commodity <- GCAM_region_ID <- Imp_t <- curr_table <- element <- item <-
+      netExp <- scaler <- value <- year <- NULL   # silence package check notes
+
```

Comment:
Definitely not urgent. But, I wouldn't mind understanding what you are doing here.